config.py

```diff
@@ -0,0 +1,24 @@
+"""Flask configuration."""
+import os
+from os import environ, path
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
+load_dotenv(path.join(basedir, '.env'))
+
+
+class Config:
+    TESTING = False
+    DEBUG = False
+
+
+class ProdConfig(Config):
+    FLASK_ENV = 'production'
+    PATH_FOR_INITIAL_DATA = os.environ.get("PATH_FOR_INITIAL_DATA") or path.join(basedir, 'restaurants.json')
```

Comment:
In `ProdConfig` there should be not predefined value, just `os.environ.get("PATH_FOR_INITIAL_DATA")` should be enough.

Also, FYI:
The second parameter of `os.environ.get` - `default_value`, you can do like:
```python
os.environ.get("PATH_FOR_INITIAL_DATA", path.join(basedir, 'restaurants.json'))
```

Reply:
fixed